build-sdt/build-standard-targets.xml

```diff
@@ -0,0 +1,560 @@
+					</fileset>
+				</copy>
+			</then>
+		</if>
+	</target>
+
+	<target name="generate-from-wsdl" if="wsdl.exists" depends="clean-src-gen-work" description="Generate CXF files from wsdl">
+		<!-- TODO - find way of collecting up all wsdl files using wild card -->
+        <java classname="org.apache.cxf.tools.wsdlto.WSDLToJava" fork="true" jvm="${java1.7-exec}">
+            <arg value="-d" />
+            <arg value="${src-gen-work.dir}" />
+            <arg value="-frontend" />
+            <arg value="jaxws21" />
+            <arg value="-b" />
+            <arg value="${resources.dir}/jaxb-bindings.xml" />
+            <arg value="${resources.dir}/wsdl/SdtEndpoint.wsdl" />
+            <classpath>
+                <path refid="cxf.classpath" />
+            </classpath>
+        </java>
+        <java classname="org.apache.cxf.tools.wsdlto.WSDLToJava" fork="true" jvm="${java1.7-exec}">
+            <arg value="-d" />
+            <arg value="${src-gen-work.dir}" />
+            <arg value="-frontend" />
+            <arg value="jaxws21" />
+            <arg value="-b" />
+            <arg value="${resources.dir}/jaxb-bindings.xml" />
+            <arg value="${resources.dir}/wsdl/SdtInternalEndpoint.wsdl" />
+            <classpath>
+                <path refid="cxf.classpath" />
+            </classpath>
+        </java>
+        <java classname="org.apache.cxf.tools.wsdlto.WSDLToJava" fork="true" jvm="${java1.7-exec}">
+            <arg value="-d" />
+            <arg value="${src-gen-work.dir}" />
+            <arg value="-frontend" />
+            <arg value="jaxws21" />
+            <arg value="-b" />
+            <arg value="${resources.dir}/jaxb-bindings.xml" />
+            <arg value="${resources.dir}/wsdl/TargetAppInternalEndpoint.wsdl" />
+            <classpath>
+                <path refid="cxf.classpath" />
+            </classpath>
+        </java>
+	</target>
+	
+   <target name="generate-from-wsdl-client" if="wsdl.exists" depends="clean-src-gen-work" description="Generate CXF files from wsdl">
+        <java classname="org.apache.cxf.tools.wsdlto.WSDLToJava" fork="true" jvm="${java1.7-exec}">
+            <arg value="-d" />
+            <arg value="${src-gen-work.dir}" />
+            <arg value="-frontend" />
+            <arg value="jaxws21" />
+            <arg value="-b" />
+            <arg value="${resources.dir}/jaxb-bindings.xml" />
+            <arg value="${resources.dir}/wsdl/SdtEndpointDist.wsdl" />
+            <classpath>
+                <path refid="cxf.classpath" />
+            </classpath>
+        </java>
+    </target>
+
+
+	<!-- Compile all src/main/java source -->
+	<target name="-compile-src-gen" if="src-gen.exists" depends="init" description="Compile src directories">
+		<javac executable="${javac1.7-exec}" fork="yes" target="${java1.7}" source="${java1.7}" debug="${javac.debug}" srcdir="${src-gen.dir}" destdir="${classes.dir}" classpathref="build.src-gen.classpath" includeAntRuntime="false">
```

Comment:
Upgrade ${java1.7} and ${java1.7-exec} params

Reply:
we are not using this module at all, will be deleted later once we are happy with our build process.